### DISCUSSION TITLE
[월드 1.1.7] 제페토 스크립트에서의 상속

### DISCUSSION THREAD

**Discussion by smilejsu82** [naverz/zepeto-studio-kor#626] [5. Q&A]
월드 1.1.7 입니다 

다음과 같이 상속 해서 사용하고 싶습니다 
테스트 결과 안되는거 같은데 되는건지 확인좀 부탁 해요 ~ 

![image](https://user-images.githubusercontent.com/50188795/163679622-8876cad5-9532-41bb-b899-be4bbc830554.png)

![image](https://user-images.githubusercontent.com/50188795/163679538-3e84c222-5d0c-45b2-ba49-2f88f2c13792.png)
![image](https://user-images.githubusercontent.com/50188795/163679548-4c7e9d8b-969d-4125-b272-9d7493ec597e.png)

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 상속 기능에 대한 질문이라면 잘 동작하고 있습니다.
만약 안된다면 호출 부분과 에러 부분을 같이 첨부해주시면 도움이 될 것 같습니다.

**Reply by smilejsu82** [reply]
> 안녕하세요, 상속 기능에 대한 질문이라면 잘 동작하고 있습니다. 만약 안된다면 호출 부분과 에러 부분을 같이 첨부해주시면 도움이 될 것 같습니다.

감사합니다 잘 동작 하네요 👍 
월드가 업데이트 되었군요 
[월드 1.2.0] 입니다 
![image](https://user-images.githubusercontent.com/50188795/163752428-5f320cea-ab75-4a72-b3b8-3cd8aca70e4b.png)

![image](https://user-images.githubusercontent.com/50188795/163752297-774ea765-e4ae-474d-bdbe-2bdc26500372.png)
![image](https://user-images.githubusercontent.com/50188795/163752316-dfc695ea-eb2f-4326-874e-181accfe35e6.png)
![image](https://user-images.githubusercontent.com/50188795/163752326-33473bb2-ac77-4dde-8ecd-a34f03732377.png)
![image](https://user-images.githubusercontent.com/50188795/163752342-b78ec8ee-5b10-4275-b193-0b5cb1dfe9cd.png)
![image](https://user-images.githubusercontent.com/50188795/163752346-4bd3fade-54ce-4d06-9242-ba85744388c1.png)
